README.md

```diff
@@ -14,18 +12,12 @@
 
 - **NOTE**: If you have issues with any of the set up, check the [Troubleshooting section](#Troubleshooting).
 
-- Before anything else, request access to the AND central gitlab. This can often take a couple of days to go through, so you can complete the rest of the environment setup whilst waiting for access:
-  - Head to the _Solarwinds service desk_ from Okta and search for _'gitlab'_ in the search bar.
-  - Click on _'Request gitlab access'_ and fill in the details.
-  - Submit the ticket and check back on it every few hours in case one of the service desk team have added any comments asking for more info.
+- Before anything else, request access to the Club Murray Organisation on GitHub. This can be requested from Owners on [this list](https://github.com/orgs/and-murray/people) - you may need another ANDi on the project to access this list on GitHub for you so you know who to ask to grant you access.
 - Follow the steps on the [React Native docs](https://reactnative.dev/docs/environment-setup) to get setup. You want to follow the _'Installing dependencies'_ instructions under the React Native CLI Quickstart tab for both iOS and Android.
   - For the Ruby section, [RVM](https://rvm.io/) is recommended to make it easy to manage Ruby versions.
 
 ## Cloning the project
-
-- Once you have gitlab access, ask the team to be added to the project.
-- You'll need to have the AND vpn running in order to access the gitlab, which can be downloaded and setup from okta (look for _AND vpn_). With the VPN running, you should be able to access the [project](https://git.core.and-digital.com/murray/murray-booking).
-- Clone the project onto your machine using either SSH or HTTPS. I recommend using SSH, which requires you to setup an [SSH key](https://git.core.and-digital.com/-/profile/keys). Once you've generated an SSH key and added it to your gitlab, you should be able to clone the project in the usual fashion.
+- Clone the project onto your machine using either SSH or HTTPS. We recommend using SSH, which requires you to setup an [SSH key](https://github.com/settings/keys). Once you've [generated an SSH key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) and [added it to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account), you should be able to clone the project in the usual fashion.
 - Downloading [Git Credential Manager](https://github.com/git-ecosystem/git-credential-manager) is recommended to help in resolving any authentication issues that may arise.
```

Comment:
MINOR 
I would remove this

Reply:
👍 Best to remove the cloning section entirely then? Because the only other item in this section is the git credential manager recommendation

Reply:
Sorry I just meant, remove the cred manager thing